compiler/src/dotty/tools/dotc/core/tasty/AttributePickler.scala

```diff
@@ -0,0 +1,25 @@
+package dotty.tools.dotc.core.tasty
+
+import dotty.tools.dotc.ast.{tpd, untpd}
+
+import dotty.tools.tasty.TastyBuffer
+import dotty.tools.tasty.TastyFormat.AttributesSection
+
+import java.nio.charset.StandardCharsets
+
+object AttributePickler:
+
+  def pickleAttributes(
+    attributes: List[String],
+    pickler: TastyPickler,
+    buf: TastyBuffer): Unit =
+      if attributes != Nil then
+        pickler.newSection(AttributesSection, buf)
+        for attribute <- attributes do
+          val bytes = attribute.getBytes(StandardCharsets.UTF_8).nn
```

Comment:
Could we assert here that `attribute` contains ASCII letters and digits? This way, if we later want to add key-value-based attributes, we won't have to introduce weird hacks.

Reply:
I will restrict them to `[a-zA-Z0-9]+`. Should I include `-` or `_`?

Reply:
`[a-zA-Z0-9]+` is enough if we use PascalCase.